### PR TITLE
Feature/ Updates 8-27

### PR DESCRIPTION
**Pull request by inkblotty**

### CODE REVIEW

File: .github/workflows/test_and_lint.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+name: Test and Lint
```
Comment (cooljasonmelton) [inline .github/workflows/test_and_lint.yml:1]: 42 files changed!! lol

Comment (inkblotty) [reply .github/workflows/test_and_lint.yml:1]: Yeah type changes lead to a *ton* of updates 😅

Comment (cooljasonmelton) [reply .github/workflows/test_and_lint.yml:1]: lol never heard of that

File: theme/src/typography/XX.ts [hunk 2]
```diff
@@ -3,16 +3,68 @@ import { Typography } from "@lb/shared";
...
   },
   headings: {
-    main: 'Arial',
-    bold: 'Arial',
-  },
-  alternate: {
-    main: 'Arial',
-    bold: 'Arial',
+    h1: {
+      family: 'Arial',
```
Comment (cooljasonmelton) [inline theme/src/typography/XX.ts:33]: arial is good default because people will recognize right away that the font is wrong lol

**Comment by cooljasonmelton** [comment]
was gonna approve but you beat me!

**Comment by inkblotty** [comment]
> was gonna approve but you beat me!

Lol sorry. I wanted to get things in a clean state in `main` before I sign off for the day! @cooljasonmelton

**Pull request merged**